docs/guides/email/02-basic-email-system.md

```diff
@@ -500,21 +527,21 @@ smtps     inet  n       -       n       -       -       smtpd
   -o smtpd_recipient_restrictions=
   -o smtpd_relay_restrictions=permit_sasl_authenticated,reject
   -o milter_macro_daemon_name=ORIGINATING
-...
 ```
 
-Finally execute the `systemctl restart postfix.service` command. At this point, the configuration of postfix is over.
+Finally execute the `systemctl restart postfix.service` command. At this point, the configuration of Postfix is over.
 
 ### Install and configure `dovecot`
 
 ```bash
-Shell(192.168.100.6) > dnf config-manager --enable devel && dnf -y install dovecot dovecot-devel dovecot-mysql
+dnf config-manager --enable devel
+dnf -y install dovecot dovecot-devel dovecot-mysql
 ```
 
-Without changing any files, the original directory structure is as follows:
+Without changing any files, the default Dovecot directory structure is as follows:
 
 ```bash
-Shell(192.168.100.6) > tree /etc/dovecot/
+Shell(192.168.100.7) > tree /etc/dovecot/
```

Comment:
@jimcat8  :

-  what is the correct IP that should appear here -  192.168.100.6 (original) OR 192.168.100.7 (new) ?

Reply:
192.168.100.6